### PR TITLE
Workloads SAPVirtualInstance New GA API 2024-09-01

### PR DESCRIPTION
**Pull request by jaskisin**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

**This PR is a copy of https://github.com/Azure/azure-rest-api-specs-pr/pull/18927 from private repo which is merged.
The new version 2024-09-01 introduced in this PR is a copy of earlier preview version 2023-10-01-preview along with conversion to typespec.
No new properties/apis have been introduced.

### CODE REVIEW

**Review by ajaygit158 — APPROVED** [review]

**Comment by ajaygit158** [comment]
Known tooling issue. Approved the suppression.

File: specification/workloads/Workloads.Operations.Management/main.tsp [hunk 1]
```diff
@@ -36,4 +36,12 @@ enum Versions {
   @useDependency(Azure.Core.Versions.v1_0_Preview_1)
   @armCommonTypesVersion(Azure.ResourceManager.CommonTypes.Versions.v3)
   v2023_10_01_preview: "2023-10-01-preview",
+
```
Comment (MaryGao) [inline specification/workloads/Workloads.Operations.Management/main.tsp:39]: Do we want to include operations list in SAPVirtualInstance? If so, we should include relevant files directly in SAPVirtualInstance side. You could ask for help in [typespec channel](https://teams.microsoft.com/l/channel/19%3A906c1efbbec54dc8949ac736633e6bdf%40thread.skype/TypeSpec%20Discussion?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).

/cc @josefree

Comment (ajaygit158) [reply specification/workloads/Workloads.Operations.Management/main.tsp:39]: Hi @MaryGao 

Our readme has reference to both SAPVirtualInstance as well as operations.
https://github.com/Azure/azure-rest-api-specs/blob/8ad58020cfafc5ace71d85b31272acf4d6b694d3/specification/workloads/resource-manager/Microsoft.Workloads/SAPVirtualInstance/readme.md?plain=1#L40C3-L42C52

Is readme not used anymore to generate the SDKs ?

Comment (MaryGao) [reply specification/workloads/Workloads.Operations.Management/main.tsp:39]: @ajaygit158 We generated from TypeSpec not from swagger. /cc @josefree

**Pull request merged**